model/registry/container.yaml

```diff
@@ -95,3 +95,23 @@ groups:
         brief: >
           Container labels, `<key>` being the label name, the value being the label value.
         examples: [ 'container.label.app=nginx' ]
+  - id: registry.container.cpu
```

Comment:
Do we really need a new attribute group for this in the registry or can we just add it to the above, flat list of attributes?

Reply:
I wasn't sure if I should semantically group this one with the rest but based on the "flat list" principle it does make sense. Changed with [f946775](https://github.com/open-telemetry/semantic-conventions/pull/282/commits/f9467758cd6e44e539d8b0f22704dc1f4f1368ca).